api/storage/client.go

```diff
@@ -147,3 +147,16 @@ func (c *Client) AddToUnit(storages []params.StorageAddParams) ([]params.ErrorRe
 	}
 	return out.Results, nil
 }
+
+// Destroy destroys specified storage entities.
+func (c *Client) Destroy(tags []names.Tag) ([]params.ErrorResult, error) {
```

Comment:
I cannot see any benefit in this. If you can, please explain. As it is, the tag types disambiguate the volume and filesystem IDs. Moving parseStorageIds to the facade is problematic, because we need to also interpret the result structures by prepending filesystem- or volume- to the tag IDs.